scripts/scr_flavor/scr_flavor.gml

```diff
@@ -135,6 +135,10 @@ function scr_flavor(argument0, argument1, argument2, argument3, argument4) {
 	    if (argument1.dudes_num[targeh]=1) then p1=string(argument3)+" "+string(wepp)+" fire upon the "+string(duhs)+".";
 	    if (argument1.dudes_num[targeh]>1) then p1=string(argument3)+" "+string(wepp)+" fire on the "+string(duhs)+" ranks.";
 	}
+	if (wepp="Stalker Pattern Bolter") and (solod=false){flavored=1;
+	    if (argument1.dudes_num[targeh]=1) then p1=string(argument3)+" "+string(wepp)+" fire upon the "+string(duhs)+".";
```

Comment:
I suggest to use template strings, for example;

`if (argument1.dudes_num[targeh]=1) then p1=$"{string(argument3)} {string(wepp)} fire upon the {string(duhs)}.";`

way easier to read and edit